src/autocomplete.ts

```diff
@@ -15,6 +15,7 @@ export default class Autocomplete {
   feedback: HTMLElement | null
   autoselectEnabled: boolean
   clientOptions: NodeListOf<HTMLElement> | null
+  clearButton: HTMLElement | null
```

Comment:
we can be more strict and use `HTMLButtonElement`

Reply:
Technically, someone can add the ID to another element like an anchor or a span. 🤔  I would like to keep it this way just in case, but I'm open to other ideas.